### ISSUE TITLE
Incorrect assertions in HTMLInputElement tests?

### ISSUE THREAD

**Issue by droeh** [web-platform-tests/wpt#17792]
I'm adding support for the `capture` attribute to Gecko and I'm a bit confused by two assertions:
https://github.com/web-platform-tests/wpt/blob/ef6e66c99b4001ea3383274acb280da559099d82/custom-elements/reactions/HTMLInputElement.html#L23
https://github.com/web-platform-tests/wpt/blob/ef6e66c99b4001ea3383274acb280da559099d82/custom-elements/reactions/HTMLInputElement.html#L46

Shouldn't the `oldValue` in both of these assertions be `null` rather than an empty string? It doesn't seem to me that `capture` is getting set anywhere prior in order for the old value to be non-null.

**Comment by gsnedders** [comment]
cc/ @Honry who wrote these tests, admittedly years ago. (also @domenic and @annevk in case they know)

**Comment by annevk** [comment]
The other problem with this test is the `if ('capture' in HTMLInputElement.prototype) {` conditional. If this is a tentative test it should be marked as such, but it shouldn't contain optional parts.

**Comment by domenic** [comment]
Yes, I think it should be null instead of empty string. And also the file should be renamed to something like `HTMLInputElement-capture`, and the conditional-ness removed.

**Comment by Honry** [comment]
I just drop the PR https://github.com/web-platform-tests/wpt/pull/20330, PTAL, thanks!
